config/test.json

```diff
@@ -81,8 +81,9 @@
 		"marketplace-personal-premium": false,
 		"me/account-close": true,
 		"me/vat-details": true,
-		"migration-flow/new-migration-instructions-step": false,
 		"migration-flow/enable-migration-assistant": true,
+		"migration-flow/introductory-offer": true,
```

Comment:
For now, only the tests feature flag is enabled. So we can write the tests for the new feature.